packages/office-ui-fabric-react/src/components/Coachmark/Beak/Beak.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  css,
+  classNamesFunction
+} from '../../../Utilities';
+import { IBeakProps } from './Beak.types';
+import { getStyles, IBeakStyles, IBeakStylesProps } from './Beak.styles';
+import { getComboBoxOptionClassNames } from 'src/components/ComboBox/ComboBox.classNames';
+
+export interface IBeakState {
+  left: string | null;
+  top: string | null;
+}
+
+export class Beak extends BaseComponent<IBeakProps, IBeakState> {
+  private _beakElement: HTMLElement;
+
+  constructor(props: IBeakProps) {
+    super(props);
+  }
+
+  public componentDidMount(): void {
+    const beakComputedStyles = window.getComputedStyle(this._beakElement);
+    this.setState({
+      left: beakComputedStyles.left,
+      top: beakComputedStyles.top
+    });
+  }
+
+  public render(): JSX.Element {
+    const {
+      height = 18,
+      width = 18,
+      left,
+      top
+    } = this.props;
+
+    const getClassNames = classNamesFunction<IBeakStylesProps, IBeakStyles>();
+    const classNames = getClassNames(getStyles, {
+      left: this.props.left,
+      top: this.props.top,
+      height: height + 'px',
+      width: width + 'px'
+    });
+
+    const pointOne = width / 2 + ',' + 0;
+    const pointTwo = width + ',' + height;
+    const pointThree = 0 + ',' + height;
```

Comment:
This will not quite wind up with an equilateral triangle. Right now the sides will be longer than the bottom leading it to be pointier than the current beak, even though the width is the same as its height.

Reply:
For Coachmark we want the beak to be a bit wider so it looks better when it's collapsed in a circle. This code gives some flexibility to the size of the triangle in case it needs to be tweaked. I'm open for suggestions on creating a friendly interface for the shape of the triangle. This was my first attempt at that.